go/vt/vtgate/engine/fake_vcursor_test.go

```diff
@@ -826,6 +830,13 @@ func (f *loggingVCursor) ReadTransaction(_ context.Context, _ string) (*querypb.
 	return f.transactionStatusOutput, nil
 }
 
+func (f *loggingVCursor) UnresolvedTransactions(_ context.Context, _ string) ([]*querypb.TransactionMetadata, error) {
+	if f.resultErr != nil {
+		return nil, f.resultErr
+	}
+	return nil, nil
+}
+
 // SQLParser implements VCursor
```

Comment:
if there is no engine test, this method implementation is not required.

Reply:
Added more engine test that are using this.